cert-manager.tf

```diff
@@ -0,0 +1,56 @@
+resource "kubernetes_namespace" "cert_manager" {
```

Comment:
I think it will much easier to look if you put the namespaces in a file like `namespaces.tf`, but this is my view.

Reply:
This namespace will be only used for cert-manager, so keeping them grouped with the operator seems easier and cleaner, same with the others.